ubuntu.json

```diff
@@ -153,6 +154,13 @@
   ],
   "post-processors": [
     {
+      "type": "compress",
+      "only": ["vmware-iso"],
+      "output": "{{.BuilderType}}_{{user `version`}}.tar.gz",
+      "keep_input_artifact": false,
+      "compression_level": 1
+    },
+    {
```

Comment:
This change is not in the description of the PR and I doubt that it is intended.